android-components/components/browser/state/src/test/java/mozilla/components/browser/state/action/ContentActionTest.kt

```diff
@@ -852,4 +852,21 @@ class ContentActionTest {
 
         assertNull(tab.content.appIntent)
     }
+
+    @Test
```

Comment:
I put form data tests in `https://searchfox.org/mozilla-mobile/source/firefox-android/android-components/components/browser/state/src/test/java/mozilla/components/browser/state/engine/middleware/SessionPrioritizationMiddlewareTest.kt#54`, I would suggest keeping them all in the same place

Reply:
Since this specific code doesn't touch `SessionPrioritizationMiddleware` at all, I don't think we should put this in the tests for it.